_scripts/configs/otel-collector-config.yaml

```diff
@@ -0,0 +1,36 @@
+receivers:
+  otlp:
+    protocols:
+      http:
+
+exporters:
+  logging:
+    loglevel: debug
+  jaeger:
+    endpoint: "jaeger.otel:14250"
+    tls:
+      insecure: true
+  googlecloud:
+    project: moz-fx-fxa-nonprod-375e
```

Comment:
Should this be a config or pickuped up automatically?

Reply:
I'll add config for this.